### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by CarlosHWolox**
## Summary
Added footer on the project

## Trello Card
https://trello.com/c/VQodRrl2/14-footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -82,6 +82,11 @@ <h2 class="tech-title">DJango</h2>
...
   </div>
+  <footer>
+    <h3 class="footer-title">We are game changers</h3>
+    <img class="footer-img" src="./assets/wolox_logo_white.svg" alt="Wolox">
+    <p class="footer-copyright">Copyright 2018. Wolox. All rights reserved.</p>
+  </footer>
 </body>
 
-</html>
+</html>
```
Comment (HarryPerez) [inline index.html]: add empty line after this

File: scss/footer.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
+footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 40px 0 10px 0;
+
+  * {
```
Comment (HarryPerez) [inline scss/footer.scss]: remember to be generic with the styles you are applying. Apply this in each style you want.

File: scss/footer.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
+footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 40px 0 10px 0;
```
Comment (HarryPerez) [inline scss/footer.scss]: Try to apply this:
captura de pantalla 2018-06-01 a la s 15 56 07

**Review by HarryPerez — CHANGES_REQUESTED** [review]

File: scss/footer.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  align-self: flex-end;
+  color: $white;
+  font-size: $text-small;
+  margin: 12px 30px;
+}
+
+.footer-img {
+  margin: 12px 10px;
+  min-width: 400px;
+
```
Comment (HarryPerez) [inline scss/footer.scss]: delete this empty space and we are done! 👍

**Review by HarryPerez — CHANGES_REQUESTED** [review]

**Review by HarryPerez — APPROVED** [review]

**Review by SKOLZ — APPROVED** [review]

**Pull request merged**